pom.xml

```diff
@@ -44,7 +44,6 @@
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-			<version>4.12</version>
```

Comment:
What version are we using?

Reply:
It's the same version that develop has right now

Reply:
No wait, nevermind. I misread the change.

Reply:
I already added it again